src/components/search/Search.jsx

```diff
@@ -123,6 +127,27 @@ const Search = () => {
   const { content_type: contentType } = refinements;
   const hasRefinements = Object.keys(refinements).filter(refinement => refinement !== 'showAll').length > 0 && (contentType !== undefined ? contentType.length > 0 : true);
 
+  if (!searchClient) {
```

Comment:
Do you feel this code path needs test coverage to assert that without a search client, highlights still render? Or are you comfortable without it?

Reply:
test added validating the search error component is correctly displayed